Source/AwsCommonRuntimeKit/AwsCommonRuntimeKit.swift

```diff
@@ -4,6 +4,7 @@ import AwsCAuth
 public struct AwsCommonRuntimeKit {
   public static let version = "@AWS_CRT_SWIFT_VERSION@"
 
+  // Todo: which library to init? Auth does init of http
```

Comment:
it's safe to init them all. They protect against repeated init

Reply:
Thanks, will fix it in a future PR. Just doing init of http & auth is confusing.